rotkehlchen/chain/evm/tokens.py

```diff
@@ -219,11 +225,11 @@ def _get_multicall_token_balances(
     def _query_chunks(
             self,
             address: ChecksumEvmAddress,
-            tokens: list[EvmToken],
+            tokens: list[EvmTokenDetectionData],
             chunk_size: int,
             call_order: list[WeightedNode],
-    ) -> dict[EvmToken, FVal]:
-        total_token_balances: dict[EvmToken, FVal] = defaultdict(FVal)
+    ) -> dict[Asset, FVal]:
```

Comment:
same comment here for the docstring. But since this is a function down in the call line probablyjust needs a small 1-liner